tests/ui/borrowck/cloning-in-async-block-121547.stderr

```diff
@@ -0,0 +1,22 @@
+error[E0382]: use of moved value: `value`
+  --> $DIR/cloning-in-async-block-121547.rs:5:9
+   |
+LL |   async fn clone_async_block(value: String) {
+   |                              ----- move occurs because `value` has type `String`, which does not implement the `Copy` trait
+LL |       for _ in 0..10 {
+   |       -------------- inside of this loop
+LL | /         async {
+LL | |             drop(value);
+   | |                  ----- use occurs due to use in coroutine
+LL | |
+LL | |         }.await
+   | |_________^ value moved here, in previous iteration of loop
+   |
+help: consider cloning the value if the performance cost is acceptable
```

Comment:
I feel like this error message is not *right* in spirit. Really we should be saying something like "cloning the value [if this is the behavior that you intended]".

Cloning here could have different semantics than actually moving the value into the async block, for example.

Reply:
Just to make sure, are you suggesting we should change the help message in general for cloning or only for this particular case in the async block?

Reply:
I don't actually know :/ I guess this is a more general problem than what this aims to fix

Reply:
May I suggest creating an issue where we can talk about the appropriate wording? I would like to also share my opinion, however, I think this is not the right PR for that.